lib/pbench/test/unit/server/test_file_tree.py

```diff
@@ -25,11 +31,8 @@ def file_sweeper(server_config):
     # After each test case:
 
     for tree in trees:
-        for d in tree.iterdir():
-            if d.is_dir():
-                shutil.rmtree(d, ignore_errors=True)
-            else:
-                d.unlink()
+        shutil.rmtree(tree, ignore_errors=True)
+        tree.mkdir(parents=True)
```

Comment:
FYI: I tried removing this, and was forcibly reminded why I added it. The config fixtures run before we get here, and expects that the `ARCHIVE` directory exists. I might be able to force the fixture order, but it doesn't seem worthwhile.

Conversely, I could have made this more complicated by scanning the directory and unlinking each file or rmtree-ing each subdirectory: it was easier just to remove the whole thing and recreate it, so that's what I did.

Reply:
Fixtures can reference each other, but without promoting this fixture to the global `conftest.py` level (where it's overkill) in order to make `setup` (which parses the config file and needs the directories to exist) depend on it seems convoluted. (Fixtures are convoluted enough.) This hack seemed like a reasonable compromise.

In any case, it's clear I'm not going to finish the PR this year, so maybe I'll take another stab at it later. I'd hoped to get this finished so I could build other things on top of it.